src/components/MultiSuggestField/MultiSuggestField.stories.tsx

```diff
@@ -78,6 +78,40 @@ predefinedNotControlledValues.args = {
     itemLabel: (item) => item.testLabel,
 };
 
+const DeferredSelectionTemplate: StoryFn = () => {
+    const initialSelected: Array<{ testId: string; testLabel: string }> = [];
+    const [loaded, setLoaded] = useState(false);
+
+    const selected = loaded ? selectedItems : initialSelected;
+
+    const identity = useCallback((item: string): string => item, []);
+
+    return (
+        <>
+            <div>Selected items loaded: {loaded.toString()}</div>
+
+            <br />
+
+            <MultiSuggestField<string>
+                items={items.map(({ testId }) => testId)}
+                selectedItems={selected.map(({ testId }) => testId)}
+                itemId={identity}
+                itemLabel={(itemId) => items.find(({ testId }) => testId === itemId)?.testLabel ?? itemId}
+                createNewItemFromQuery={(query) => query}
+            />
+
+            <br />
+
+            <button onClick={() => setLoaded((prev) => !prev)}>Toggle selected</button>
+        </>
+    );
+};
+
+/**
+ *
+ */
+export const deferredSelection = DeferredSelectionTemplate.bind({});
```

Comment:
The "clear all" button on the field does not work for this example. Is it just the story, or is is some bug within the elemen code?

Reply:
That was a story bug, I forgot to remove the selection function from the component.
Check again please.

Also, I added tests with clearance for both controlled and uncontrolled components.